web/packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -281,6 +281,14 @@ export const formatters: Formatters = {
         rest['server_hostname'] || rest['addr.local']
       }] failed [${exitError}]`,
   },
+  [eventCodes.SCP_DISALLOWED]: {
+    type: 'scp',
+    desc: 'SCP Disallowed',
+    format: ({ user, ...rest }) =>
+      `SCP command from user [${user}] to node [${
```

Comment:
We should do our best to use consistent language in user-facing code. 

Our other events mostly take the form "user X [failed to] Y" - any reason to depart from that here?

Reply:
Nope, just an oversight, fixed